indico/modules/designer/controllers.py

```diff
@@ -172,14 +172,20 @@ def _process(self):
 
 
 class AddTemplateMixin(TargetFromURLMixin):
+    always_clonable = False
+
     def _checkProtection(self):
         if not self.target.can_manage(session.user):
             raise Forbidden
 
     def _process(self):
         form = AddTemplateForm()
+        if self.always_clonable:
+            del form.is_clonable
         if form.validate_on_submit():
-            new_template = DesignerTemplate(title=form.title.data, type=form.type.data, **self.target_dict)
+            is_clonable = form.is_clonable.data if form.is_clonable else True
```

Comment:
It works since `form.is_clonable` returns `None` in case the field doesn't exist in the form.